controls/src/main/java/org/tweetwallfx/controls/WordleSkin.java

```diff
@@ -285,17 +292,14 @@ public double minWidth(double height) {
     
     public void prepareStepMachine() {
         startupLogger.info("Prepare StepMachine");
-        StepIterator steps = new StepIterator(Arrays.asList(//UpdateCloudStep(),
-                new FadeInCloudStep(),
-                new NextTweetStep(),
-                new AddTweetToCloudStep(),
-                new CloudToCloudStep(),
-                new CloudToTweetStep(),
-                new PauseStep(),
-                new TweetToCloudStep(),
-                new CloudFadeOutStep(),
-                new ImageMosaicStep()
-        ));
+        StepIterator.Builder builder = new StepIterator.Builder();
+        try(InputStream s =  this.getClass().getResourceAsStream("/steps.json")){

```

Comment:
Bike shed question: should StepIterator configure itself based on a provided resource? In case it should this should be moved to StepIterator.